src/elements/content-sidebar/__tests__/__snapshots__/SidebarFileProperties.test.js.snap

```diff
@@ -98,6 +100,7 @@ exports[`elements/content-sidebar/SidebarFileProperties render() should render r
       className="loading-indicator-wrapper "
     >
       <ItemProperties
+        archivedAt={NaN}
```

Comment:
We should handle cases where the value isn't a number by hiding the line item.

Reply:
Fixed after conversion change